tests/route/test_route_consistency.py

```diff
@@ -11,6 +17,58 @@
 logger = logging.getLogger(__name__)
 
 
+def kill_process_by_pid(duthost, container_name, program_name, program_pid):
+    """Kills a process in the specified container by its pid.
+
+    Args:
+        duthost: Hostname of DUT.
+        container_name: A string shows container name.
+        program_name: A string shows process name.
+        program_pid: An integer represents the PID of a process.
+
+    Returns:
+        None.
+    """
+    if "20191130" in duthost.os_version:
```

Comment:
QQ. Do we need to run this test on DUTs running 2019?
Also why would the below command not work there?

Reply:
I copied this function from the other test file: https://github.com/sonic-net/sonic-mgmt/blob/master/tests/process_monitoring/test_critical_process_monitoring.py

I am not sure why it needs a special treatment for 20191130 images. I can delete it if you'd like to, the killing mechanism should be general to all Linux distributions.

Reply:
i see.. then any reason to copy this function here instead of importing it?

Reply:
importing it directly from the other test file will create a cross-tests dependency, I wasn't sure if this is allowed. Should I change the code to import it from the other test file? or should I put the function into some files in tests/common/ and then make both test cases to import the function?

Reply:
yes. better to place that function to some helper file.

Reply:
Updated